README.md

```diff
@@ -8,7 +8,13 @@ Team members:
 - [Swapnali Sonkusale]()
 - [Tejas Choudekar]()
 
-## Virtual Environment
```

Comment:
We should keep this as "Virtual Environment & Dependencies"

Reply:
Done